src/main/java/net/masterthought/cucumber/json/Result.java

```diff
@@ -27,8 +25,7 @@ public String getFormatedDuration() {
         return Util.formatDuration(duration);
     }
 
-    /** Returns error message escaped so it is properly displayed as HTML. */
     public String getErrorMessage() {
-        return StringEscapeUtils.escapeHtml(errorMessage);
+        return errorMessage;
```

Comment:
Except in a few tests that were now comparing unescaped text against escaped text.